### PR TITLE
New CI/CD for Exchange, Wrapper, Linter, and Web

### PR DESCRIPTION
**Pull request by stevenewald**
Only thing missing is CD for Linter

### CODE REVIEW

File: .github/workflows/exchange-ci.yml [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+        conan install . -s build_type=Release -b missing
+
+    - name: Setup MultiToolTask
+      if: matrix.os == 'windows-2022'
+      run: |
+        Add-Content "$env:GITHUB_ENV" 'UseMultiToolTask=true'
+        Add-Content "$env:GITHUB_ENV" 'EnforceProcessCountAcrossBuilds=true'
+
+    - name: Configure
+      shell: pwsh
```
Comment (echavemann) [inline .github/workflows/exchange-ci.yml:242]: ew

**Comment by stevenewald** [comment]
> lgtm, linter CD should be not too bad

Linter CD is already done I just haven't moved it (will do at some point)

File: .github/workflows/exchange-ci.yml [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+    branches:
+      - main
+    paths:
+      - 'exchange/**'
+
+  workflow_dispatch:
+
+# We only care about the latest revision of a PR, so cancel all previous instances.
+concurrency:
+  group: ci-${{ github.event.pull_request.number || github.ref_name }}
```
Comment (egelja) [inline .github/workflows/exchange-ci.yml:20]: this needs to have a unique value for each ci

`ci-exchange-${{ github.event.pull_request.number || github.ref_name }}`

File: exchange/Taskfile.yml [hunk 3]
```diff
@@ -69,3 +69,10 @@ tasks:
   default:
     cmds:
       - task: run
+
+
+  lint:
```
Comment (egelja) [inline exchange/Taskfile.yml:74]: lint-fix

Comment (stevenewald) [reply exchange/Taskfile.yml:74]: why

**Pull request merged**